.github/workflows/release.yml

```diff
@@ -67,6 +70,39 @@ jobs:
           if-no-files-found: error
           retention-days: 1 # Maximum Retention
 
+      - name: Copy YAML artifact to website/static/releases
```

Comment:
You could do it this way, but it would be better to put the releases into website/src/data/ccc-releases rather than website/static as this is for serving static files out of Docusaurus.  

Also, I wrote a script website/src/scripts/DownloadCCCReleases.ts which runs when docusaurus gets built (locally or on netlify) which will update the releases in the directory anyway, so we probably don't really need to create new PRs for this